tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -79,6 +80,49 @@ module.exports = {
         const isPresent = notifications.includes(element.title)
         this.assert.ok(isPresent)
       }
+    },
+    /**
+     * Perform accept action on the offered shares in the notifications
+     */
+    acceptAllSharesInNotifications: async function () {
+      await this.click('@notificationBell')
+        .waitForElementVisible('@notificationElement')
+      await this.api.elements('@notificationElement', result => {
+        for (const element in result.value) {
+          let elementIndex = parseInt(element)
+          elementIndex = elementIndex + 1
+          const acceptShares = util.format(this.elements.acceptSharesInNotification.selector, elementIndex)
+          this
+            .useXpath()
+            .waitForElementVisible(acceptShares)
+            .click(acceptShares)
+        }
+      })
+    },
+    /**
+     * Checks to assert the notification bell is not present as well as no notifications are present
+     */
+    assertNoNotifications: function () {
+      return this.waitForElementNotPresent('@notificationBell')
+        .waitForElementNotPresent('@notificationElement')
```

Comment:
Too much wait, wouldn't just be asserting the second time work?

Reply:
I think this would be useful to assert that the notification bell is absent when there are no notifications on the notification element.The test should fail when there is some bug due to which the bell is still present when there are no notifications.(or something like that).What do you think?